controller/workitem.go

```diff
@@ -123,6 +121,10 @@ func (c *WorkitemController) Update(ctx *app.UpdateWorkitemContext) error {
 	if creator == nil {
 		return jsonapi.JSONErrorResponse(ctx, errors.NewInternalError(ctx, errs.New("work item doesn't have creator")))
 	}
+	creatorID, err := uuid.FromString(creator.(string))
```

Comment:
Please, never ever do a type conversion without checking for the result. Otherwise we have a panic.

```suggestion
	creatorIDStr, ok := creator.(string)
	if !ok {
		return jsonapi.JSONErrorResponse(ctx, errs.Errorf("failed to convert user to string: %+v (%[1]T)", creator))
	}
	creatorID, err := uuid.FromString(creatorIDStr)
```

Reply:
@kwk thanks for pointing out. addressed it here 
41bc004